### ISSUE TITLE
iproute2 & cpu_arch

### ISSUE THREAD

**Issue by martdah** [internetarchive/hind#1]
I really enjoyed your talks, thank you!

    iproute2 install added to provide /sbin/ip (nomad chucks its toys out the pram if missing)
    extract hardcoded amd64 and make it a little more flexible and open to docker building container for different 
architectures

[Dockerfile.zip](https://github.com/internetarchive/hind/files/9226821/Dockerfile.zip)
[install-docker-ce.sh.zip](https://github.com/internetarchive/hind/files/9226822/install-docker-ce.sh.zip)

**Comment by traceypooh** [comment]
ooh!   I like this. 
lol at first I thought you were spam since you attached zips.
but maybe I don't have "allow pull requests" open for this repo or something?

interested in making a PR and getting "credit" for being contributor #2 to this project?  :wink: 

thanks!

**Comment by martdah** [comment]
I would be happy to do so, I did try to send a PR, then created an issue when that failed. The zips were due to github not accepting "Dockerfile" and stating that a zip was fine.
This project is quite interesting to me, I'm hoping it can become a default dev setup, we run a mac fleet and as you may or may not know, setting hashi tooling on a mac, using docker-desktop is not only delicate, it's a flippin mare!
more recently exacerbated by apples new silicon.

**Comment by traceypooh** [comment]
Fantastic!

Can confirm the package seems needed now, too, for the current arch setup.

Could you… fork this and make your changes, push, and _then_ make a PR that I can doublecheck and approve and merge?
(Or maybe you already tried that?)

I tend to use gitlab in my work day-to-day more… so am sometimes a bit less fluent with GitHub.
But I *think* that might work as a flow for you to get your changes in and you’ll get commit credits (yay) and all that, too.

I feel you on the Mac thing.  Am about to get a M2 MB Air in just over a day so will be _much more_ pulled into the apple silicon drama, lol.

-Tracey

**Comment by traceypooh** [comment]
I see your fork! :) 

what do you think about:
`dpkg --print-architecture`

I found for example that 
```
wget "https://caddyserver.com/api/download?os=linux&arch=$(uname -p)" 
```

didn't work (comes up with `x86_64`

but 
```
wget "https://caddyserver.com/api/download?os=linux&arch=$(dpkg --print-architecture)"
```
comes up w/ the same `amd64` string as before (at least on my linux on x86 VMs at work).

what do you think?

**Comment by traceypooh** [comment]
fixed by #2 - thanks @martdah !
